sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py

```diff
@@ -989,3 +990,79 @@ def update_transcription(
             **kwargs
         )
         self._call_media_client.update_transcription(self._call_connection_id, update_transcription_request)
+
+    @distributed_trace
+    def hold(
+        self,
+        target_participant: 'CommunicationIdentifier',
+        *,
+        play_source: Optional[Union['FileSource', 'TextSource', 'SsmlSource']] = None,
+        operation_context: Optional[str] = None,
+        operation_callback_uri: Optional[str] = None,
+        **kwargs
+    )-> None:
+        """Play media to specific participant(s) in this call.
+
+        :param target_participant: The participant being added.
+        :type target_participant: ~azure.communication.callautomation.CommunicationIdentifier
+        :param play_source: A PlaySource representing the source to play.
+        :type play_source: ~azure.communication.callautomation.FileSource or
+         ~azure.communication.callautomation.TextSource or
+         ~azure.communication.callautomation.SsmlSource
+        :keyword operation_context: Value that can be used to track this call and its associated events.
+        :paramtype operation_context: str or None
+        :keyword operation_callback_url: Set a callback URL that overrides the default callback URL set
+         by CreateCall/AnswerCall for this operation.
+         This setup is per-action. If this is not set, the default callback URL set by
+         CreateCall/AnswerCall will be used.
+        :paramtype operation_callback_url: str or None
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+        play_source_single: Optional[Union['FileSource', 'TextSource', 'SsmlSource']] = None
+        if isinstance(play_source, list):
```

Comment:
is there a way to prevent the user from passing a list?

Reply:
> is there a way to prevent the user from passing a list?

Currently i don't see any other options. as these are different class